Main.java

```diff
@@ -9,7 +10,38 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // scanner for user input 
+    Scanner input = new Scanner(System.in);
+
+    //asks user for input
+    System.out.println("Please enter a number to play FizzBuzz");
+    
+    //user input
+    int number = input.nextInt();
+
+    int Fizz = (number%3);
+    int Buzz = (number%5);
+
+    if (Fizz == 0 && Buzz == 0){
+    System.out.println("you should say FizzBuzz");
+    }
+    
+    else if (Fizz == 0){
+      System.out.println("you should say Fizz");
+    }
+    else if (Buzz == 0){
+      System.out.println("you should say Buzz");
+    }
+  else if (Buzz != 0 && Fizz != 0){
+    System.out.println("you should say "+ number);
+  }
```

Comment:
Watch your indentation. If there is code inside of an if statement, it should be TABBED over an extra bit to show it is inside of the if-statement.